lib/octicons_react/package.json

```diff
@@ -28,6 +29,7 @@
     "primer"
   ],
   "dependencies": {
+    "@types/react": "^16.4.6",
```

Comment:
I think this can be moved to a `devDependency` as it's not needed at runtime (or for non-TS usage)

Reply:
It is needed at runtime, though — the `index.d.ts` file directly `import`s it.

Reply:
This is how `react-testing-library` is [setup](https://github.com/kentcdodds/react-testing-library/blob/master/package.json#L43-L44) and I didn't have an issue with using that recently in Desktop:

```
  "devDependencies": {
    "@types/react-dom": "^16.0.6",
    ...
```

Reply:
Also, feel free to relax the version needed here - many packages just do `"@types/react": "*"` unless they depend on specific APIs.